core/src/main/scala/org/apache/spark/scheduler/local/LocalSchedulerBackend.scala

```diff
@@ -178,4 +188,23 @@ private[spark] class LocalSchedulerBackend(
     }
   }
 
+  private def updateDelegationTokens(): Unit = {
+    if (UserGroupInformation.isSecurityEnabled()) {
+      val delegationTokenManager =
```

Comment:
Since we don't store the reference of the `HadoopDelegationTokenManager` how does this renew the tokens? Or renewal is not considered?

Reply:
The `renewal` would run at `HadoopDelegationTokenManager` if the conditions are matched.

Reply:
Since we added as `var` now it works, thanks.